### PR TITLE
Don't track GitHub deployments

### PR DESCRIPTION
**Pull request by mrfyda**

### CODE REVIEW

File: docs/one-click-integrations.md [hunk 1]
```diff
@@ -62,10 +62,15 @@ The Pulse GitHub integration supports two strategies to detect and measure deplo
...
 
     -   The set of changes that belong to a deployment is the list of commits between the tag of that deployment and the previous tag. Because of this, Pulse discards:
-        
+
         -   The first SemVer tag in the repository since there is no previous tag to compare with.
         -   Any tag that does not have a common ancestor (commit) with its previous tag, since Pulse cannot obtain the changes between them.
 
+-   **Don't track automatically**
+
+    -   Pulse will not try to infer deployments from any GitHub event.
```
Comment (prcr) [inline docs/one-click-integrations.md:71]: Suggestion to simplify the information we want to convey here:

```suggestion
    -   Pulse won't detect deployments using GitHub events.
```

**Review by prcr — COMMENTED** [review]
:muscle:

**Comment by prcr** [comment]
I did a few more edits and updated the screenshot. Here's a preview of the end result with the changes highlighted:

image

**Review by claudiacodacy — APPROVED** [review]
Looking good Paulo! Good job! 😎

**Review by rtfpessoa — APPROVED** [review]

**Pull request merged**